apps/projects/app/components/Content/Settings.js

```diff
@@ -502,4 +548,25 @@ const Column = styled.div`
   max-width: ${35 * GU}px;
 `
 
+const IconContainer = styled.button`
+  ${unselectable};
+  all: unset;
+  color: ${({ disabled, theme }) => (disabled ? theme.disabled : theme.contentSecondary)};
+  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
+  display: flex;
+  justify-content: center;
+  :hover {
+    color: ${({ disabled, theme }) => disabled ? theme.disabled : theme.contentBorderActive};
+  }
+  :active {
+    color: ${({ disabled, theme }) => (disabled ? theme.disabled : theme.accent)};
```

Comment:
Let's wrap the whole set of hover & active styles in a conditional. We don't need them if `disabled`. So:

```js
${p => !p.disabled && css`
  :hover { ... }
  :active { ... }
`}

Reply:
Does this work for you?
```
${({ disabled, theme }) => (disabled ? `
      color: ${theme.disabled};
      cursor: not-allowed;
    ` : `
      color: ${theme.contentSecondary};
      cursor: pointer;
      :hover {
        color: ${theme.surfaceOpened};
      }
      :active {
        color: ${theme.accent};
      }
    `)
}
```